common/src/main/java/io/pravega/schemaregistry/common/NameUtil.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.schemaregistry.common;
+
+public class NameUtil {
+    /**
+     * Extracts the name from the fully qualified type name. Name represents the last token after ".". 
+     * If the qualified name does not contain "." then the name is same as qualified name. 
+     * 
+     * @param qualifiedName qualified name to extract name from. 
+     * @return extracted name. 
+     */
+    public static String extractName(String qualifiedName) {
+        int nameStart = qualifiedName.lastIndexOf(".");
+        return qualifiedName.substring(nameStart + 1);
+    }
+
+    /**
+     * Extracts name and the prefix qualifier before the name. Name represents the last token after ".".
+     * Qualifier is the prefix before the name. 
+     * If the qualified name does not contain "." then the name is same as qualified name and qualifier is empty string. 
+     * 
+     * @param qualifiedName qualified name to extract tokens from. 
+     * @return an array containing name at index 0 and qualifier at index 1. 
+     */
+    public static String[] extractNameAndQualifier(String qualifiedName) {
+        int nameStart = qualifiedName.lastIndexOf(".");
```

Comment:
Reuse `extractName()` here

Reply:
yes, i made changes in the branch where i am managing the complete implentation (for-test).. in copying the changes over, the "common" module was perhaps not included as even hashutil changes were not included. sorry. i have fixed it now. thanks.